src/Demo/localApiServer.ts

```diff
@@ -26,64 +26,45 @@ export default class LocalApiServer {
   }
 
   public extensionCustomGet() {
-    const suggestions = this.store.suggestions.
-      filter((s) => s.type !== 'default').
-      map((s) => {
-        return { id: s.suggestion, text: s.suggestion };
-      });
 
-    const hotkeys = this.store.hotkeys.
-      map((h) => {
-        return { id: h.suggestion, text: h.expandedSuggestion, trigger: h.suggestion, type: h.type };
-      });
+    const snippets = this.store.snippets;
```

Comment:
I'd  make a copy of the object here to prevent modifying the original object from this.store